builds/releasenotes.md

```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
   - Images in the scene are not using thumbnails (EVC-513)
 
 Version 0.4.0 (WIP)
-  - Improvements
-    - ?
-  - Other Fixes and Changes
-    - ?
-
+  - Conversion Improvements
+    - Fixed issue with resetting some conversion types
+    - Increased support for additional types of FBX files
+    - Added drop down to set all spaces in a conversion rather than doing it individually
+    - Convert CMD takes a global offset now (`-globaloffset X Y Z`)
+    - Added Earth Centered, Earth Fixed support to conversion
+    - Fixed some issues with estimation of resolution for conversion
+    - Added support for additional SRID codes from customers and the community
+  - Improvements to SLPK
+    - Loading is ordered to focus on things closer to the camera
```

Comment:
This is kind of an awkward way of saying "Loading prioritizes objects closer to the camera".